charts/prometheus-elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 6.0.0
+version: 6.0.1
```

A new feature as in this change needs raising the minor version number.

```suggestion
version: 6.1.0
```

---

@zeritti fixed it